mmedit/models/backbones/sr_backbones/ttsr_net.py

```diff
@@ -9,6 +10,8 @@
 # Use partial to specify some default arguments
 _norm_conv_layer = partial(
     build_conv_layer, dict(type='Conv2d'), kernel_size=3, padding=1)
+_bottleneck_layer = partial(
```

Comment:
I'm not sure if **one** conv could be called a "bottleneck"

Reply:
Also double check if normalization in included

Reply:
Renamed to `_conv1x1_layer`